.gitignore

```diff
@@ -1,5 +1,7 @@
 .vscode/
-wurst/war3map.j
-objectEditingOutput/*
-compiled.j.txt
+/_build
 
+*.w3x
```

Comment:
Can we ignore w3x in _build instead? I don't want all w3x files globally ignored.